util/chplenv/printchplenv.py

```diff
@@ -88,10 +88,11 @@
     ChapelEnv('CHPL_LAUNCHER', LAUNCHER | DEFAULT, 'launch'),
     ChapelEnv('CHPL_TIMERS', RUNTIME | LAUNCHER | DEFAULT, 'tmr'),
     ChapelEnv('CHPL_UNWIND', RUNTIME | LAUNCHER | DEFAULT, 'unwind'),
-    ChapelEnv('CHPL_HOST_MEM', INTERNAL, 'hostmem'),
-    ChapelEnv('CHPL_TARGET_MEM', INTERNAL, 'tgtmem'),
+    ChapelEnv('CHPL_HOST_MEM', COMPILER, 'hostmem'),
+    ChapelEnv('CHPL_TARGET_MEM', INTERNAL, 'mem'),
```

Comment:
Why the change from `tgtmem` to `mem`?

Reply:
My thought was that since the intention for `CHPL_TARGET_MEM` to replace `CHPL_MEM`, they should have the same name. I don't think this has an effect either way today